codalab/lib/interactive_session.py

```diff
@@ -175,10 +190,9 @@ def _get_container_name(self):
     def _construct_final_command(self):
         try:
             candidate_commands = self._get_bash_history()
-        except Exception:
+        except Exception as e:
             print(
-                'The history of bash commands could not be retrieved at path: %s'
-                % InteractiveSession._BASH_HISTORY_CONTAINER_PATH,
+                'The history of bash commands could not be retrieved at path: {}'.format(e),
```

Comment:
Did you mean to include `InteractiveSession._BASH_HISTORY_CONTAINER_PATH` in this error message as well?